src/reducers/match.js

```diff
@@ -25,7 +26,7 @@ export default (state = initialState, action) => {
         ...state,
         loading: false,
         error: false,
-        match: action.payload,
+        match: renderMatch(action.payload),
```

Comment:
I think this kind of transformation should not happen in the reducer, but rather the action. That way, if something goes wrong in renderMatch in the future, we can understand and know about that by inspecting the action it is dispatching in redux debugging tools.